diff --git a/python/CMakeLists.txt b/python/CMakeLists.txt
index c817fc2..623f158 100644
--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -41,7 +41,7 @@ file(
 )
 
 add_custom_target(PyPackageBuild ALL
-    COMMAND ${Python3_EXECUTABLE} -m build --sdist
+    COMMAND ${Python3_EXECUTABLE} -m build --sdist --no-isolation
     COMMAND ${Python3_EXECUTABLE} -m pip install .
     WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
     COMMENT "Building python sdist package"
